mono/mini/trace.h

```diff
@@ -24,11 +24,11 @@ typedef struct {
 	void *data, *data2;
 } MonoTraceOperation;
 
-struct MonoTraceSpec {
+typedef struct MonoTraceSpec {
```

Comment:
This should just be `typedef struct { ...`.

Reply:
The struct name is required for forward declaration. I'll just cancel this change - the typedef already exists in mini.h so I'll include mini.h where necessary